app/containers/message/index.tsx

```diff
@@ -97,8 +97,20 @@ class MessageContainer extends React.Component<IMessageContainerProps, IMessageC
 
 	shouldComponentUpdate(nextProps: IMessageContainerProps, nextState: IMessageContainerState) {
 		const { isManualUnignored } = this.state;
-		const { threadBadgeColor, isIgnored, highlighted, previousItem, autoTranslateRoom, autoTranslateLanguage, isBeingEdited } =
-			this.props;
+		const {
+			threadBadgeColor,
+			separator,
+			isIgnored,
+			highlighted,
+			previousItem,
+			autoTranslateRoom,
+			autoTranslateLanguage,
+			isBeingEdited
+		} = this.props;
+
+		if (nextProps.separator !== separator) {
```

Comment:
Although it works, it's causing unnecessary re-renders.

The separator is a ReactElement and by doing `nextProps.separator !== separator` we're doing a shallow comparison of two objects, which is going to always return true.

Can you try moving separator to children instead of a normal prop?

```
<Message>
  <Separator>
</Message>
```

Please read this for more info: https://www.developerway.com/posts/react-elements-children-parents